src/codeu/chat/common/ConversationHeader.java

```diff
@@ -63,4 +63,21 @@ public ConversationHeader(Uuid id, Uuid owner, Time creation, String title) {
     this.title = title;
 
   }
+
+  @Override
+  public boolean equals(Object toCompare)
```

Comment:
Remember that if you override `equals()`, you should also override `hashcode()`. More info here: https://stackoverflow.com/questions/2265503/why-do-i-need-to-override-the-equals-and-hashcode-methods-in-java

Basically, if you'll ever use these objects in a `HashMap` or a `HashSet`, they'll first use `hashcode()` to find items, and then they'll use `equals()` to compare them. You don't want two items to be equal (according to your new `equals()` function) but **not** equal according to `hashcode()`.